packaging/linux/build_and_push_packages.sh

```diff
@@ -90,14 +90,12 @@ release_prerelease() {
 
   json_tmp=`mktemp`
   echo "Writing version into JSON to $json_tmp"
-  cat > "$json_tmp" <<END
-{
-"version": "$version",
-"name": "v$version",
-"description": "Latest Linux release",
-"type": 0
-}
-END
+
+  echo "Loading release tool"
+  "$client_dir/packaging/goinstall.sh" "github.com/keybase/release"
+  release_bin="$GOPATH/bin/release"
```

Comment:
This is also done in s3_index.sh. But that's ok, right? `goinstall.sh` is intended to be safe to run multiple times?